plugins/techdocs-node/api-report.md

```diff
@@ -18,6 +17,11 @@ import { ScmIntegrationRegistry } from '@backstage/integration';
 import { UrlReader } from '@backstage/backend-common';
 import { Writable } from 'stream';
 
+// @public
+export interface ContainerRunner {
```

Comment:
Should we deprecate the old exports from the `backend-common` as well?

Reply:
I'm not sure since we are not providing any alternative 🤔

Reply:
My understanding is that we would like to deprecate and not provide a replacement in the new system as we do not want be the maintainers of the container runner. So I think the decision is recommend plugins to keep their own implementation of it. But let's double-check with @Rugvip

Reply:
To repeat back where we landed on this: we deprecate and remove the container runner types and implementations from `backend-common`. In TechDocs it becomes a completely internal implementation, i.e. we should not be exporting these types here. The scaffolder we can discuss separately, but we'll keep the container runner implementation around internally in some form for the cookiecutter and rails images.

Reply:
but then we need to mark this as breaking. Added #c674ffc1641f248666c1924d4b2a9df43f09d0fe let me know if it makes sense